README.md

```diff
@@ -1,96 +1,132 @@
+
+```hcl
+module "postman_test_lambda" {
+  source = "github.com/byu-oit/terraform-aws-postman-test-lambda?ref=v3.0.0"
+  app_name = "simple-example"
+  postman_collections = [
+    {
+      collection = "terraform-aws-postman-test-lambda-example.postman_collection.json"
+      environment = "terraform-aws-postman-test-lambda-env.postman_environment.json"
     }
-    ```
-   Using this method allows you to not have to export your collection and commit the JSON file to your repo.
-   
-   **Note:** The postman collection/environment must be viewable by the postman account tied to the API key you provide.
-   
-   **Note 2:** Make sure your postman collection/environment names are unique, otherwise you will get an error if the postman API finds more than 1 collection/environment with the same name.
-   
-   **DON'T** hard code your postman API key, treat it like all other secrets.
-
-Then add your lambda function_name to the CodeDeploy lifecycle hook you want the postman tests to run on.
-For instance, if you're using the [fargate-api module](https://github.com/byu-oit/terraform-aws-fargate-api):
+  ]
+  role_permissions_boundary_arn = data.aws_ssm_parameter.role_permissions_boundary_arn.value
+}
+```
+
+You can specify multiple collections and environments to run in the lambda function. The function will run the
+collections in order.
+
+You can run collections/environments from local json files or using the [Postman API](#using-the-postman-api).
+
+**Note:** When [using the Postman API](#using-the-postman-api): the postman collections/environments must be viewable by
+the postman account tied to the API key you provide.
+
+**DON'T** hard code your postman API key, treat it like all other secrets.
+
+Then add your lambda function_name to the CodeDeploy lifecycle hook you want the postman tests to run on. For instance,
+if you're using the [fargate-api module](https://github.com/byu-oit/terraform-aws-fargate-api):
+
 ```hcl
 # ... postman-test-lambda module
 
 module "fargate_api" {
-  source = "github.com/byu-oit/terraform-aws-fargate-api?ref=" # latest version
+  source = "github.com/byu-oit/terraform-aws-fargate-api?ref="
+  # latest version
   # .. all other variables
   codedeploy_lifecycle_hooks = {
-    BeforeInstall         = null
-    AfterInstall          = null
+    BeforeInstall = null
+    AfterInstall = null
     AfterAllowTestTraffic = module.postman_test_lambda.lambda_function.function_name
-    BeforeAllowTraffic    = null
-    AfterAllowTraffic     = null
+    BeforeAllowTraffic = null
+    AfterAllowTraffic = null
   }
 }
 ```
+
 Or if you're using the [lambda-api module](https://github.com/byu-oit/terraform-aws-lambda-api):
+
 ```hcl
 # ... postman-test-lambda module
 
 module "lambda_api" {
-  source = "github.com/byu-oit/terraform-aws-lambda-api?ref=" # latest version
+  source = "github.com/byu-oit/terraform-aws-lambda-api?ref="
+  # latest version
```

Comment:
did this mean to be on a new line?

Reply:
Good catch. I'll tweak that and also try to merge in William's vpc code in this before merging

Reply:
The example above this also had the same new line, so I wasn't sure if it was on purpose or not.

Reply:
Should be fixed